interface/patient_file/summary/stats.php

```diff
@@ -199,15 +199,19 @@ function load_location( location ) {
                 $rowclass = "noreturn";
             }
 
+
+
             echo " <tr class='text $rowclass;'>\n";
 
             //turn allergies red and bold and show the reaction (if exist)
             if ($key == "allergy") {
                 $reaction = "";
                 if (!empty($row['reaction'])) {
-                    $reaction = " (" . $row['reaction'] . ")";
+                    $reaction= " (" .xlt(ucfirst(str_replace("_"," ",$row['reaction']))) . ")";
                 }
 
+                $row['title'] = xlt(ucfirst(str_replace("_"," ",$row['title'])));
```

Comment:
Aren't users typing this title into a text field. And if so, shouldn't they be entering it via their language?

Reply:
o.k. removed that row .